apps/print/frontend/src/utils.ts

```diff
@@ -25,35 +24,3 @@ export function getAvailableLanguages(election: Election): LanguageCode[] {
     .filter((lang) => lang !== undefined)
     .sort(sortLanguages);
 }
-
-// sortBallotPrintCounts sort order: totalCount, precinctOrSplitName, partyName, languageCode
-export function sortBallotPrintCounts(
```

Comment:
Moved sorting to backend store function so ballot print counts are sorted consistently regardless of use (i.e. pdf report vs. ui)